### PR TITLE
Added upgrade warning for users about taxonomy settings options

### PR DESCRIPTION
**Pull request by kgaikwad**
@ares, @tbrisker 
Please check if below note is good to add under audits section.

### CODE REVIEW

**Review by tbrisker — COMMENTED** [review]
I think this might be better placed in the upgrade warning section (1.2) of the manual

**Comment by kgaikwad** [comment]
@tbrisker,
Moved note under upgrade warning section.
Thank you!

File: _includes/manuals/1.20/1.2_release_notes.md [hunk 1]
```diff
@@ -500,6 +500,7 @@ values as part of their default, should now be configured via the UI, CLI or API
 them can be set directly in `settings.yaml` or via the installer.
 * Access to `@host.params['param_name']` in templates, which has been deprecated several versions ago, has been removed. Templates can access host parameters using the `host_param('param_name')` macro instead.
 * Operating System `pxe_files`, `pxe_prefix`, `kernel`, `initrd`, `bootfile`, `boot_files_uri`  methods now expect a Medium Provider as their first argument. Please update any templates accordingly. See the [redmine issue](https://projects.theforeman.org/issues/19389) and [associated changes](https://github.com/theforeman/foreman/commit/ef1bbc264acfe0864bd24e7ce54e6b37dcda37ad) for further details.
+* The audit history doesn't display correctly if disabled locations and organizations options in your settings.yaml. These options are now deprecated, see [the redmine issue](https://projects.theforeman.org/issues/24800) for more details.
```
Comment (tbrisker) [inline _includes/manuals/1.20/1.2_release_notes.md]:
```suggestion
* The audit history doesn't display correctly if locations or organizations have been disabled in your settings.yaml. These options are now deprecated, see [the redmine issue](https://projects.theforeman.org/issues/24800) for more details.
```

**Review by tbrisker — COMMENTED** [review]
tiny wording change suggestion inline

**Review by tbrisker — APPROVED** [review]
Thanks @kgaikwad !

**Pull request merged**